code/game/turfs/simulated/floor/plating/misc_plating.dm

```diff
@@ -195,7 +195,7 @@
 	icon_state = "smooth"
 	smooth = SMOOTH_MORE | SMOOTH_BORDER
 	canSmoothWith = list(/turf/open/floor/plating/ice/smooth, /turf/open/floor/plating/ice)
-	/turf/open/floor/plating/ice/colder
```

Comment:
??

Reply:
redundant see its the same line as below